odoo/addons/base/res/res_partner.py

```diff
@@ -34,9 +34,10 @@
 }
 
 ADDRESS_FIELDS = ('street', 'street2', 'zip', 'city', 'state_id', 'country_id')
+
 @api.model
 def _lang_get(self):
-    return self.env['res.lang'].get_installed()
+    return self.env['res.lang'].with_context(prefetch_fields=False).get_installed()
```

Comment:
Original method is getting `lang.code, lang.name` then 2 queries will be executed similar to:
`SELECT code FROM res_lang WHERE id=%d`
`SELECT name FROM res_lang WHERE id=%d`

Using prefetch just one query is executed:
`SELECT * FROM res_lang WHERE id=%d`

Maybe, you like to pre-fetch the fields manually running a read before to call the method.

In fact, maybe you need this change from get_installed method instead.
https://github.com/odoo/odoo/blob/8f2267b512efe068c9ce261de8480cb4b5843187/odoo/addons/base/models/res_lang.py#L209-L214

Since,
`langs = self.with_context(active_test=True).search([])`

Using
`langs = self.with_context(active_test=True, prefetch_fields=False).search([])`
`langs.read(['code', 'name'])  # fetch fields used a line below`


Offtopic note: Why the original method is using a `sorted` instead of a `order` parameter for search method.

Reply:
Thanks @moylop260 , I will try your suggestions